tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -1970,6 +1963,6 @@ public boolean isValueLocked() {
      * @return true or false
      */
     public boolean canFlagOnLosing() {
-        return valueIsLocked || (requiredVotes == HIGH_BAR);
+        return valueIsLocked || (getRequiredVotes() == HIGH_BAR);
```

Comment:
This is the only change that's not refactoring or testing. If getRequiredVotes() returns 8 that's not high enough to flag.